core/src/main/scala/org/apache/spark/SparkException.scala

```diff
@@ -29,7 +29,8 @@ class SparkException(
     cause: Throwable,
     errorClass: Option[String],
     errorSubClass: Option[String],
-    messageParameters: Array[String])
+    messageParameters: Array[String],
+    context: Array[QueryContext] = Array.empty)
```

Comment:
@MaxGekk As you mentioned, you plan to add context in `SparkException`. Is this part looking good to you?

Reply:
Yep.